### ISSUE TITLE
Post-release notifications

### ISSUE THREAD

**Issue by nevadascout** [HvyIndustries/crane#91]
After a user has updated to the latest version of Crane or installed Crane for the first time, do we need to display a help message linking to the release notes and telling them how to get started?

Also, do we need to be able to force a rebuild of intellisense caches? If our release changes the way the treebuilder works (as 0.2 does), then we need to be able to invalidate a cache. Maybe we include the Crane version number in our hash calculation? eg. `projectname_version` is the string that gets hashed - then this will be done automatically when we increment the Crane version.

**Overview**
- [x] Store the version number in the crane folder in a `version` file
- [x] Compare the version number on launch. If different (or not exists):
  - [x] Display a message informing the user of the update with a link to the release notes/wiki
  - [x] Delete all the cache folders so they are re-created
  - [x] Update the version file to the current version
- [x] Add command to delete all caches

- [x] Option to rebuild project on update
- [x] Stuck on indexing after "install"

**Comment by TheColorRed** [comment]
The cache currently does a `JSON.stringify(workspaceTree)` and then uses gzip to zip it up. What we could also do, is put in the Crane version that the cache was built with.

For example:

```js
var workspaceTree = {
    version: "1.7.0",
    // Not sure if this is the best date format or not, but it is the same as a database timestamp
    created: "2016-06-18 14:24:55", 
    updated: "2016-06-21 10:11:12",
    tree: [
        /* Object 1*/,
        /* Object 2*/
    ]
}
```

I don't know if we want to link to changes or not, when the user searches for an to install or update, there is that **README** link on the right hand side of the extension. If your are thinking that the link will be different than the **Getting Started** or **Release Notes**, then maybe would be a good idea...

**Comment by nevadascout** [comment]
I was going to take a slightly simpler approach. I think it could be very confusing/annoying if you have a cache for a project and it's 'invalidated' by an update, causing a user to be given incorrect or broken suggestions which may lead to a bug report.

Instead we should just take the easy option and delete all the caches when we release an update. This avoids this problem completely and only causes a slightly longer load time after an update.

**Comment by TheColorRed** [comment]
If we delete all caches on an update, then I think we should also have a command to delete all caches/projects as well.

**Comment by nevadascout** [comment]
Resolved in #101
